AzureTable/Microsoft.DataTransfer.AzureTable.Wpf/XamlResources.xaml

```diff
@@ -10,6 +10,9 @@
 
   <system:String x:Key="ConnectionStringKey">Connection String</system:String>
   <system:String x:Key="ConnectionStringSummaryKey">Connection String:</system:String>
+  <system:String x:Key="LocationModeKey">Location Mode</system:String>
+  <system:String x:Key="LocationModeSummaryKey">Location Mode:</system:String>
+  <system:String x:Key="LocationModeDescriptionKey">Choose which Location mode to use. Unless you have secondary azure storage configured do NOT change this from PrimaryOnly</system:String>
```

Comment:
I'd use "Location Mode" (capital 'M'). Then official name is "Azure Storage" (capitalized again). And, lastly, if you will update LocationMode to show as human readable text, then you can update it to "Primary Only" (with space).